Dockerfile.cypress

```diff
@@ -0,0 +1,68 @@
+##################################################
+#
+# cypress image
+FROM cypress/included:13.10.0
+
+##################################################
+#
+# go backend build
```

Comment:
Thinking about it, we wont need to build backend in the this image, since we are testing agains live cluster.